SRC/Pages/OVWObjectViewer.Page.al

```diff
@@ -82,6 +82,22 @@ page 50130 "OVW Object Viewer"
                     RunApplicationObject();
                 end;
             }
+
+            action(ExportToExcel)
+            {
+                ApplicationArea = All;
+                Caption = 'Export To Excel';
+                Image = Excel;
+                Promoted = true;
+                PromotedCategory = Process;
+                PromotedIsBig = true;
```

Comment:
PromotedIsBig is an obsolete option and can be left out.

Also: please add a tooltip to your action.